1-exercises/B-while-loop/exercise.js

```diff
@@ -6,9 +6,17 @@
 */
 
 function evenNumbers(n) {
-    // TODO
+    let i = 0;
+    let arr = [];
```

Comment:
This is a good way to do it if we were looking to return a string, how might you modify your method to return a comma separated string?